packages/react-native/ReactCommon/react/renderer/graphics/Transform.h

```diff
@@ -50,6 +51,23 @@ struct TransformOperation {
   Float z;
 };
 
+struct TransformOrigin {
+  std::array<YGValue, 3> origin;
```

Comment:
Do you reckon it might make more sense to make separate `YGValue x`, `YGValue y` and `double z` properties - since `z` cannot be a percentage

Reply:
yeah, made xy an array and z as float. The array makes it easier to iterate. https://github.com/facebook/react-native/pull/38559/commits/302a9aaf283a2d05de01ea314e996fb848c89bcb